package.json

```diff
@@ -31,47 +31,46 @@
   "homepage": "https://github.com/esbb48/Kizuna#readme",
   "dependencies": {
     "babel-polyfill": "^6.9.1",
-    "grommet": "^0.6.10",
+    "extract-text-webpack-plugin": "^v2.0.0-beta.4",
+    "grommet": "^1.0.1",
     "react": "^15.3.0",
     "react-dom": "^15.3.0",
-    "react-hot-loader": "^3.0.0-beta.1",
+    "react-hot-loader": "^3.0.0-beta.6",
     "react-redux": "^4.4.5",
-    "react-router": "^2.6.1",
+    "react-router": "^3.0.0",
     "react-router-redux": "^4.0.5",
     "redux": "^3.5.2",
     "redux-action": "^1.0.0",
-    "redux-saga": "^0.11.0"
+    "redux-saga": "^0.13.0"
   },
   "devDependencies": {
-    "autoprefixer": "^6.3.7",
     "babel-core": "^6.11.4",
-    "babel-eslint": "^6.1.2",
+    "babel-eslint": "^7.1.1",
     "babel-loader": "^6.2.4",
     "babel-plugin-transform-object-rest-spread": "^6.8.0",
     "babel-preset-es2015": "^6.9.0",
     "babel-preset-react": "^6.11.1",
     "eslint": "^3.2.2",
-    "eslint-config-airbnb": "^10.0.0",
-    "eslint-plugin-import": "^1.12.0",
-    "eslint-plugin-jsx-a11y": "^2.0.1",
+    "eslint-config-airbnb": "^13.0.0",
+    "eslint-plugin-import": "^2.2.0",
+    "eslint-plugin-jsx-a11y": "^2.2.3",
     "eslint-plugin-react": "^6.0.0",
-    "extract-text-webpack-plugin": "^1.0.1",
-    "html-webpack-plugin": "^2.22.0",
+    "file-loader": "^0.9.0",
+    "html-webpack-plugin": "^2.24.1",
     "mocha": "^3.0.2",
     "node-sass": "^3.8.0",
-    "postcss-loader": "^0.10.1",
-    "precss": "^1.4.0",
+    "postcss-loader": "^1.2.0",
     "sass-loader": "^4.0.0",
     "should": "^11.1.0",
     "stylelint": "^7.1.0",
-    "stylelint-config-standard": "^12.0.0",
+    "stylelint-config-standard": "^15.0.0",
     "url-loader": "^0.5.7",
-    "webpack": "^1.13.1",
-    "webpack-dev-server": "^1.14.1",
+    "webpack": "^2.1.0-beta.27",
+    "webpack-dev-server": "^2.1.0-beta.12",
     "webpack-md5-hash": "0.0.5"
   },
   "engines": {
-    "node": "v6.2.0",
-    "npm": "3.8.9"
+    "node": "v7.2.0",
```

Comment:
hmm okay.

I use v7.  because The project is not for job and we can  it's more relaxed.